ui/app/src/components/DashboardList/DashboardList.tsx

```diff
@@ -163,7 +162,7 @@ export function DashboardList(props: DashboardListProperties) {
       ></DashboardDataGrid>
       <Box>
         {targetedDashboard && (
-          <Box>
+          <>
```

Comment:
I think you can just remove this wrapper instead of changing it to fragment here

Reply:
Nope, you can't remove it 😋